das-dashboard/backend/shared/utils/remote_scp.py

```
@@ -0,0 +1,137 @@
+                detail=str(error),
+            ) from error
+        except OSError as error:
+            raise RemoteSshConnectionError(
+                f"SSH connection failed for {host}.",
+                detail=str(error),
+            ) from error
+
+        return ssh
+
+    def get_remote_home(self, ssh: SSHClient) -> str:
+        stdin, stdout, stderr = ssh.exec_command("echo $HOME")
```

Pylint (4.0.6)

[warning] Unused variable 'stdin'

(W0612)